examples/pytorch/eeg-gcnn/EEGGraphDataset.py

```diff
@@ -0,0 +1,145 @@
+import torch
+import numpy as np
+import math
+from dgl.data import DGLDataset
+import dgl
+
+
+class EEGGraphDataset(DGLDataset):
+    """ Build graph, treat all nodes as the same type
+        Parameters
+        ----------
+        x: edge weights of 8-node complete graph
+            There are 1 x 64 edges
+        y: labels (diseased/healthy)
+        num_nodes: the number of nodes of the graph. In our case, it is 8.
+
+        Output
+        ------
+        a complete 8-node DGLGraph with node features and edge weights
+    """
+    def __init__(self, x, y, num_nodes, indices, transform=None):
+        # CAUTION - epochs and labels are memory-mapped, used as if they are in RAM.
+        self.epochs = x
+        self.labels = y
+        self.indices = indices
+        self.transform = transform
+        self.num_nodes = num_nodes
+        # num_sensors = num_nodes = N = 8
+        # NOTE: this order decides the node index, keep consistent!
+        self.ch_names = [
+            "F7-F3",
+            "F8-F4",
+            "T7-C3",
+            "T8-C4",
+            "P7-P3",
+            "P8-P4",
+            "O1-P3",
+            "O2-P4"
+        ]
+
+        # in the 10-10 system, in between the 2 10-20 electrodes in ch_names, used for calculating edge weights
+        self.ref_names = [
+            "F5",
+            "F6",
+            "C5",
+            "C6",
+            "P5",
+            "P6",
+            # CAUTION: mapping has exceptional case
+            # "PO3", # not available in standard_1010.csv, using O1
+            # "PO4"  # not available in standard_1010.csv, using O2
```

Comment:
L49-L51 are a bit confusing, either we remove them or we need to provide a better explanation.